nautobot/extras/tables.py

```diff
@@ -877,8 +877,10 @@ class JobResultTable(BaseTable):
         linkify=True,
         verbose_name="Scheduled Job",
     )
-    actions = tables.TemplateColumn(
-        template_code="""
+    actions = ButtonsColumn(
+        JobResult,
+        buttons=("delete",),
+        prepend_template="""
```

Comment:
Can we put this block into a CONSTANT as well ,something like JOB_RESULT_BUTTON?

Reply:
Sure, can do.